extension/src/contentscript/index.ts

```diff
@@ -1,13 +1,25 @@
-console.log(`Loaded into page`);
-
 import { browser } from "webextension-polyfill-ts";
 
+console.log(`[CS] Loading WatchWithMe Extension content script`);
+
 const csPort = browser.runtime.connect(undefined, { name: "PORT-CS" });
-csPort.postMessage("This is the content script");
+csPort.postMessage({
+    type: "debugMessage",
+    message: "[CS] This is the content script",
+});
+
+const roomId = new URLSearchParams(window.location.search).get("roomId");
```

Comment:
I think i'd be better to use a constant file to story the query param name.

Reply:
I don't understand what you mean, let's maybe talk about it on our next standup

Reply:
I meant to say store and not story. 

constants.ts
```
export const ROOM_ID_QUERY
```js

and then in index.ts
```js
const roomId = new URLSearchParams(window.location.search).get(ROOM_ID_QUERY);
```

Reply:
I find this a bit overkill, through I understand the point. It will be used both as a query param, db identifier, websocket param, etc. so I think it's enough to leave it as a string, or we'd need to find a more generic name for the variable

Reply:
@mourtisma maybe you can do this refactoring in your next feature with websocket